docs/02-guides/server-side-events/create-an-event.mdx

```diff
@@ -7,9 +7,119 @@ sidebar_position: 1
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+## What is an event emitter
 
-- Create an event emitter
-- Emit an event from your code
+The events are dispatched with what we call Event Emitter. The role of the event emitter is to push events to the Server event pipeline to be processed by the server.
 
-:::
+In this guide, we will cover the creation of an emitter and how to implement them in your pages, we will create an Emitter that Emit an event when a customer reach a page called FAQ.
```

Comment:
**issue:** This sentence does not has anything to do with what an event emitter is really;

**suggestion:** Update the guide's description with this sentence, and remove it from this section.